BundleClient/app/src/main/java/net/discdd/bundleclient/GrpcReceiveTask.java

```diff
@@ -60,12 +62,12 @@ class GrpcReceiveTask {
         this.activity = activity;
     }
 
-    public void executeInBackground(String port, String host) {
+    public void executeInBackground(String host, String port) {
```

Comment:
we should port an int so that we don't mix this up. Or perhaps even better use InetSocketAddress.